server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesResponse.java

```diff
@@ -241,6 +241,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
+        if (indexResponses.size() > 0) {
+            return "FieldCapabilitiesResponse{unmerged}";
```

Comment:
Not great but we shouldn't throw on `toString` ever so we have to have some breakout here for the un-merged case that doesn't allow `toXContent`.